### DISCUSSION TITLE
question about async-shell-command completion

### DISCUSSION THREAD

**Discussion by morhaham** [minad/vertico#537] [Q&A] [answered]
Is it possible to show candidates in a list like with find-file?
Currently they are shown in the default menu.
<img width="1641" alt="image" src="https://github.com/user-attachments/assets/cfdaff31-1077-46bd-9d00-d6b8448cb22e">

**Comment by minad** [comment, marked as answer]
See https://github.com/minad/vertico?tab=readme-ov-file#completion-at-point-and-completion-in-region
